lib/get-last-release.js

```diff
@@ -30,7 +30,9 @@ module.exports = ({branch, options: {tagFormat}}, {before} = {}) => {
   const [{version, gitTag, channels} = {}] = branch.tags
     .filter(
       (tag) =>
-        ((branch.type === 'prerelease' && tag.channels.some((channel) => isSameChannel(branch.channel, channel))) ||
+        ((branch.type === 'prerelease' &&
+          tag.channels.some((channel) => isSameChannel(branch.channel, channel)) &&
+          semver.parse(tag.version).prerelease.includes(branch.name)) ||
```

Comment:
Is it correct to use `branch.name` here? My concern is what if the branch is configured to use a different prerelease identifier for the version.

Would this make more sense?

```suggestion
          semver.parse(tag.version).prerelease.includes(branch.prerelease)) ||
```

Reply:
Thanks, good catch!  I totally missed the fact that the `prerelease` property could be anything else than a `boolean`, as that's the only way it is used in the docs and in my usecase.

I'll change it next week when I'll have access to my laptop again.

Reply:
I changed the implementation accordingly, only had to add a test-case to check that it still would work if the prerelease-value was `true` instead of a `string` value, that's why the test might look messier than before.